src/app/components/planet-details/planet-details.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { PlanetMockServiceService } from 'src/app/services/planet-mock-service.service';
+import { PlanetService } from 'src/app/services/planet.service';
+
+import { PlanetDetailsComponent } from './planet-details.component';
+
+describe('PlanetComponent', () => {
+  let component: PlanetDetailsComponent;
+  let fixture: ComponentFixture<PlanetDetailsComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ PlanetDetailsComponent ],
+      providers: [ { provide: PlanetService, useClass: PlanetMockServiceService }]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PlanetDetailsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  // it('should render the h1 as Planets in a Galexy far far away...', () => {
```

Comment:
If its commented it out, YAGNI. https://en.wikipedia.org/wiki/You_aren't_gonna_need_it

Just a thought :)